### DISCUSSION TITLE
speedata Publisher Pro

### DISCUSSION THREAD

**Discussion by pgundlach** [speedata/publisher#436] [Announcements]
## New stable version in October

There will be a new version of speedata Publisher on the first of October (planned, so it might be delayed). 

I plan to release version 4.12 as a regular successor to 4.10 (released in July) with the state of the current 4.11.8 version and a few patches that have not merged yet.

## New Pro plan

This will be the start of a new “plan”: There will be a speedata Publisher standard plan (free to use) and a speedata Publisher professional plan aimed at business users.

The idea is to have features that are aimed at professional users in the Pro plan, such as spot colors, the server mode, QR codes and similar features. I also will open up the speedata Publisher SAAS API for professional users. The list of features is not fixed yet, but will be on release. Professional users are also eligible for e-mail support.

## speedata Publisher API

The SAAS speedata Publisher API has been around for a few years now, but is not been used much. The API will get an update and will be included in the Pro plan.

## No license change

The speedata Publisher is still licensed under the terms of the AGPL, nothing changed in this respect. All features will still be in the code tree and open for anyone.

**Comment by pr-apes** [comment]
@pgundlach,

just to confirm, there will be two kind of binaries (professional and public), but the main code will be available here (excepting probably the code for the API).

Will it also be a delay in the release of public binaries (or code) when compared to their professional counterparts?

Many thanks for your help.

**Comment by pgundlach** [comment]
Yes, all code will be here (Pro and Standard) and the binaries will released at the same time. I will continue to have only a single code base (mirrored here) and there will be no delay generating the binaries (only the delay that macOS binaries are generated on a different machine and uploaded after the Windows and Linux binaries).

The code for the API might be released one day but this is just some glue code calling the publisher within a docker file.

I hope that this answers your question.
